docs: update certificate rotation warning (#1782)
stackhpc/stackhpc-kayobe-config: doc/source/configuration/openbao.rst

Rotating OpenBao certificate on the overcloud hosts
---------------------------------------------------

> The certificate for the overcloud OpenBao has an expiry time of one or two years.
> This will be problematic if anything needs to interact with the OpenBao API such as issuing new certificates or Barbican integration.

Reviewer: If the default is one year, we should just say one year here.
I do think we should bump it to 2 years in Epoxy though
```suggestion
The certificate for the overcloud OpenBao has an expiry time of one year by default.
Certificate expiry will be problematic if anything needs to interact with the OpenBao API such as issuing new certificates or Barbican integration.
```

Author: Once this merges the default will be 2 years. https://github.com/stackhpc/stackhpc-kayobe-config/pull/1775

Reviewer: Well, that complicates things.
It would be nice to have slightly more clarity than just "one or two years". I think the best way to do that is just merge https://github.com/stackhpc/stackhpc-kayobe-config/pull/1775 first, then just say something like "certificates created since xxx will expire after 2 years, before it was 1 year"

Author: The operator should know how long a certificate is valid for. Simply merge #1775 won't changing anything unless operators reconfigure vault/openbao and then generate new certificates.

Reviewer: > The operator should know how long a certificate is valid for.

These docs should be where the operator looks when they _don't_ know how long a certificate is valid for. We have plenty of clients and junior engineers who might not know how to check that sort of thing.

> Simply merge https://github.com/stackhpc/stackhpc-kayobe-config/pull/1775 won't changing anything unless operators reconfigure vault/openbao and then generate new certificates.

That's a good point. How about instead we just show people how to check:
```suggestion
The certificate for the overcloud OpenBao has an expiry time of one or two years depending on when was created.
This will be problematic if anything needs to interact with the OpenBao API such as issuing new certificates or Barbican integration.

Use the following command to check the expiry date:

.. code-block::

    openssl foo...
```

Reviewer: Great minds :brain:
